service-admin/src/App/src/Form/UserFind.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace App\Form;
+
+use App\Validator;
+use App\Filter\StandardInput as StandardInputFilter;
+use Laminas\Filter;
+use Laminas\Form\Element\Text;
+use Laminas\Form\Element\Hidden;
+use Laminas\InputFilter\Input;
+use Laminas\InputFilter\InputFilter;
+
+/**
+ * Class UserFind
+ * @package App\Form
+ */
+class UserFind extends AbstractForm
+{
+    /**
+     * UserFind constructor
+     *
+     * @param array $options
+     */
+    public function __construct($options = [])
+    {
+        parent::__construct(self::class, $options);
```

Comment:
Potentially remove this line

Reply:
Can't be removed as it throws a 500 error. Attempted to resolve by extending Laminas Form directly, but throws the same error. Keeping in rather than wasting too much time assessing what's up with this.